tools/restore-backups-omaha.sh

```diff
@@ -0,0 +1,272 @@
+    elif ! [ -z "$data" ] ; then
+      local az=$data
+    fi
+  done
+  unset IFS
+
+  if [ -z "pvc-a87f0c51-3eb8-48d6-94ae-a086bc4f9f79"] || [ -z "${pvc2Pod['pvc-a87f0c51-3eb8-48d6-94ae-a086bc4f9f79']}" ] || [ -z "$az" ] ; then
+    echo "Could not find all volume info for volume $1"
+    exit 1
+  fi
+
+  volume2Tags[$1]=$tagspec
+  volume2PVC[$1]=$pvc
+  volume2PV[$1]=$pv
+  volume2AZ[$1]='ca-central-1a'
+}
+
+createPVCs() {
+  logStep "Recreating PVCs (and their PVs)"
+  for pvc in ${!pvc2NewVolume[@]}; do
+    local volume=${pvc2NewVolume[$pvc]}
+    createPV pv-$pvc $volume ${volume2AZ[$volume]} ${volume2Size[$volume]}
+    createPVC $pvc pv-$pvc ${volume2Size[$volume]}
+  done
+}
+
+createPV(){
+  local name=$1
+  local vol=$2
+  local az=$3
+  local region=${az::-1}
+  local size=$4
+
+  # Remove PV if present
+  kubectl get pv $name >/dev/null 2>&1 && kubectl delete pv $name
+  echo "Creating PV $name"
+  cat <<EOF | kubectl apply -f -
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: $name
+  labels:
+    failure-domain.beta.kubernetes.io/region: $region
+    failure-domain.beta.kubernetes.io/zone: $az
+spec:
+  capacity:
+    storage: ${size}Gi
+  accessModes:
+  - ReadWriteOnce
+  awsElasticBlockStore:
+    fsType: ext4
+    volumeID: aws://$az/$vol
+  nodeAffinity:
+    required:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: failure-domain.beta.kubernetes.io/zone
+          operator: In
+          values:
+          - $az
+        - key: failure-domain.beta.kubernetes.io/region
+          operator: In
+          values:
+          - $region
+  storageClassName: gp2
+
+EOF
+}
+
+createPVC(){
+  local name=$1
+  local pv=$2
+  local size=$3
+
+  # Remove PVC if present
+  kubectl get pvc $name >/dev/null 2>&1 && kubectl delete pvc $name
+  echo "Creating pvc $name"
+  cat <<EOF | kubectl apply -f -
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: $name
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: ${size}Gi
+  storageClassName: gp2
+  volumeName: $pv
+EOF
+}
+
+deletePVCs() {
+  logStep "Deleting all PVCs (and the PVs they are bound with)"
+  for pvc in ${!pvc2Pod[@]}; do
+    local pv=`kubectl get pvc $pvc -o jsonpath='{.spec.volumeName}'`
+    echo "Retaining pv $pv"
+    kubectl patch pv $pv -p '{"spec":{"persistentVolumeReclaimPolicy":"Retain"}}'
+    echo "Deleting pvc $pvc"
+    kubectl delete pvc $pvc
+    echo "Deleting pv  $pv"
+    kubectl delete pv $pv
+  done
+}
+
+createVolumes() {
+  logStep "Creating Volumes"
+  for snap in ${!snapshot2Volume[@]}; do
+    createVolume $snap
+  done
+}
+
+scaleDown() {
+  logStep "Scaling down Statefulsets"
+  scaleDown() {
+    logStep "Scaling down Statefulsets"
+    for sts in `kubectl get deployment.apps/postgres -o name`; do
```

Comment:
The for loop might be a bit overkill now that we have only one deployment to scale